d3.wordcloud.js

```diff
@@ -63,6 +70,7 @@
         .text(function(d) {
           return d.text;
         })
+        .rotate(rotate)
```

Comment:
Why not put the default function here, and allow `rotate` to be null? Avoids having the function twice.